py-rse/publish.Rmd

```diff
@@ -112,7 +111,7 @@ and/or represents a derived quantity with high re-use potential.
 For example,
 it would be worthwhile to publish
 an estimate of the global average surface temperature derived from
-a large database of weather observations,
+a large database of weather observations (and researchers have),
```

Comment:
took me a couple of tries to parse this - maybe just say "researchers have published..."?